### DISCUSSION TITLE
Include "mouse follows window" as a repeated command

### DISCUSSION THREAD

**Discussion by NeighNeighNeigh** [rxhanson/RectanglePro-Community#232] [Ideas]
I would love to try using the repeated command feature to send my mouse along with my window. So by default Rectangle Pro leaves my mouse alone, but if I issue the same shortcut immediately after, it will send my mouse to the new window position.
<img width="494" alt="move cursor on repeated command" src="https://user-images.githubusercontent.com/4795315/221708919-1a9cfaeb-cad4-41cb-8e71-d0a5c6432f1a.png">

Often I use Rectangle Pro to move windows to my secondary display and get them out of the way, so I can keep working on my primary display. In these cases I don't want the mouse to follow. Including the above functionality will give me the option at my fingertips.

Thanks Ryan, your work makes my life better and I appreciate it daily.

**Comment by rxhanson** [comment]
Thanks for the feedback! 

I've been planning on adding in cursor actions for a while now, but had not thought of this idea, thanks! I could see doing it where the cursor is moved if you execute the shortcut again without having released the modifiers (similar to that "Reset repetitions when modifiers are released" setting). That way it's not tied to a timeout. I'll give this a go when I'm working on the cursor actions and see if I can get something good in (it might be a little while before I get to it, though).

**Reply by NeighNeighNeigh** [reply]
Of course, no rush, looking forward to seeing this feature if it does make the cut :)
